context/app/static/js/components/detailPage/ProcessedData/hooks.spec.ts

```diff
@@ -1,13 +1,13 @@
 import { CreationAction } from 'js/components/types';
 import { renderHook } from 'test-utils/functions';
-import { useSortedSearchHits, createdByCentralProcess, datasetIsPublished, usePipelineCountsInfo } from './hooks';
+import { useSortedSearchHits, createdByCentralProcess, datasetIsPublished, useAnalysesCountInfo } from './hooks';
 
 const testDatasets: {
   _id: string;
   _index: string;
   _score: number;
   _source: {
-    assay_display_name: string[];
+    assay_display_name: string;
```

Comment:
I think we should leave these types as is - in HelperPanel we're accessing `assay_display_name[0]`, so having these be consistent is preferable.

Reply:
All of these functions get their prop types from the `ProcessedDatasetInfo` interface, which defines the `assay_display_name` as a string - I think it makes sense to update these tests accordingly, but I also see your point about accessing indices in `HelperPanel`. I updated that function so that it treats `assay_display_name` as a string.

Reply:
Looking back on this because it is continuing to cause issues with testing elsewhere on the detail pages - it might make more sense to instead update the `ProcessedDatasetInfo` interface to have a string array for `assay_display_name`, so that it is more accurate to the query results. I implemented this in #3582, along with making the `pipeline` field optional (as it is absent from the new EPICs).